itsnameissteven/Brew-by-Design
# Feature/recipe page

## Is this a fix or a feature?
This is a feature
## What is the change?
a recipe page has been added. with all the details of a recipe
## What does it fix?

## Where should the reviewer start?
Review all new components
## How should this be tested?
in the browser, run a search and click on a recipe. You should be redirected to a new page.

## Relevant ScreenShots
